bootcamp/tutorials/quickstart/hybrid_demo.ipynb

```diff
@@ -0,0 +1,584 @@
+{
```

Comment:
<p>In this tutorial, we will demonstrate how to conduct hybrid search with Milvus and BGE-M3 model. BGE-M3 model can convert text into both dense and sparse vectors. Milvus supports storing both types of vectors in one collection, so that hybrid search on them can improve search result relevance. </p><p> </p><p>Milvus supports Dense, Sparse, and Hybrid retrieval methods: </p><ul><li>Dense Retrieval: Utilizes semantic context to understand the meaning behind queries.</li><li>Sparse Retrieval: Emphasizes keyword matching to find results based on specific terms. This is equivalent to full-text search.</li><li>...</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1389/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>